packages/fast-components-react-base/src/listbox/listbox.tsx

```diff
@@ -83,6 +84,20 @@ class Listbox extends Foundation<
         }
     };
 
+    /**
+     * returns all the selectable items in the provided array of children
+     */
+    public static getValidOptions = (
+        childrenAsArray: React.ReactNode[]
+    ): React.ReactNode[] => {
+        return childrenAsArray.filter((itemNode: React.ReactNode) => {
+            if (Listbox.isValidSelectedItem(itemNode as React.ReactElement<any>)) {
```

Comment:
you can just return this so `return Listbox.isValidSelectedItem(itemNode as React.ReactElement<any>);` if it might return undefined just add `!!` in front.

Reply:
yep